README.md

```diff
@@ -68,3 +102,32 @@ Most formatters have some options available that can be configured via Glaive.
 You can get a quick view of all codefmt flags by executing `:Glaive codefmt`, or
 start typing flag names and use tab completion. See `:help Glaive` for usage
 details.
+
+## Creating a New Formatter
```

Comment:
Link to https://github.com/google/vim-codefmt/wiki/Formatter-Integration-Guide, consider merging some content there instead of having it inline here.